docs/src/content/docs/snippets/grant-create.mdx

```diff
@@ -296,6 +296,10 @@ Run `node path/to/directory/index.js`.
 </TabItem>
 </Tabs>
 
+:::note[Continue object]
+The continue object is included in all grant responses and is part of the standard grant response format. It is particularly important for interactive outgoing payment grants, as it facilitates the process of obtaining explicit user consent. While the continue object is present in all grant responses, it is primarily used in the context of outgoing payments. Implementations have the option to make other grants interactive if desired.
```

Comment:
`continue` object

Since the object is included in **all** grant responses, I think this note should be moved to the overview. The overview would need to be reworked so as to not have two stacked notes.

Reply:
Also I think the third sentence repeats the first sentence a bit.